app/main/views/buyer_domains.py

```diff
@@ -0,0 +1,71 @@
+from dmapiclient.audit import AuditTypes
+from sqlalchemy.exc import IntegrityError
+from flask import jsonify, abort, request
+
+from .. import main
+from ... import db
+from ...models import BuyerEmailDomain, AuditEvent
+from ...validation import validate_buyer_email_domain_json_or_400, buyer_email_domain_approved
+from ...utils import (
+    get_json_from_request, json_has_required_keys, validate_and_return_updater_request,
+    get_valid_page_or_1, pagination_links
+)
+
+
+@main.route('/buyer-email-domain', methods=['POST'])
```

Comment:
This and the GET one should probably be plural `/buyer-email-domains`, since that's what we do with every other endpoint

Reply:
Yikes, good catch 👍

Reply:
Now done - @allait could you sanity check the corresponding API client PR? https://github.com/alphagov/digitalmarketplace-apiclient/pull/106